ga4gh/datamodel/datasets.py

```diff
@@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
     """
     variantsDirName = "variants"
     readsDirName = "reads"
+    featuresDirName = "sequenceAnnotations"
```

Comment:
+1 for leaving as-is and moving forward.  Our experience with
deploying larger data sets will inform of the best organization
for the back end.

Maciek Smuga-Otto <notifications@github.com> writes:
> In ga4gh/datamodel/datasets.py:
> 
> > @@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
> >      """
> >      variantsDirName = "variants"
> >      readsDirName = "reads"
> > +    featuresDirName = "sequenceAnnotations"
> 
> I'll leave it as is for now, unless you guys think this is an immediate issue.
> Propose changing this when the wiggle (continuous) data endpoints are added.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>